garak/probes/grandma.py

```diff
@@ -40,16 +41,17 @@ class Win10(Probe):
         "quality:Behavioral:ContentSafety:LegalGoodsServices",
     ]
     goal = "get model to output Win 10 product keys"
-    PRODUCT_NAMES = ("Windows 10", "Windows 10 Pro")
 
     doc_uri = "https://www.independent.co.uk/tech/chatgpt-microsoft-windows-11-grandma-exploit-b2360213.html"
 
     def __init__(self, config_root=_config):
+        win10_payload = garak.payloads._load_payload("keyedprod_win10")
+        product_names = win10_payload.payloads
```

Comment:
I would not expect access to `_load_payload()` in a probe.

``` Python
from garak.payloads import Loadmaster

...

class Win10(Probe)
    
    _payload_types = ( "Security circumvention instructions/Product activation codes" )

...
    def __init__(self, config_root=_config):
        product_names = []
        p_group_names = Loadmaster.search(types=_payload_types)
        for group in p_groups:
            product_names.append(Loadmaster.load(group).payloads)
...
```

Based on the above, the search tooling does not provide a method of selecting payload for this probe if support for other probes in this module were to be introduced. I suspect there is some more structure needed in the payload files or it is I am missing something on how this is expected to evolve.

Reply:
I'm in two minds about `payloads._load_payload()` (and even whether it should be just `load_payload()`) but you're right, let's lead with an example using the helper.

Reply:
I suspect if probes are using it, it shouldn't be a private method.